src/containers/Dashboard/Courses/Card.js

```diff
@@ -1,28 +1,50 @@
 import React, { Component } from "react";
+import { connect } from "react-redux";
+import { withRouter } from "react-router-dom";
 import "./Card.scss";
 
 class Card extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      courseId: ""
+    };
+  }
+
+  getCourseId(courseId, name) {
+    this.props.history.push(`course-page/${name}/${courseId}`);
+  }
+
   render() {
-    const courses = this.props.data;
-    const courseList = courses.map((course, index) => {
-      return (
-        <div className="course-name" key={index}>
-          <a href="./home">{course}</a>
-        </div>
-      );
-    });
     return (
       <div className="courses-card">
         <div className="card shadow-lg">
           <div className="card-body">
             <div className="header">
               <h5 className="card-title">{this.props.title}</h5>
             </div>
-            <div className="card-content">{courseList}</div>
+            <div className="card-content">
+              {this.props.data.map((course, index) => (
+                <div
+                  className="course-name"
+                  key={index}
+                  onClick={() => this.getCourseId(course.courseId, course.name)}
```

Comment:
@Humnaa why is this method name is `getCourseID` ?

Reply:
I have renamed it getCourseById